.github/workflows/main.yml

```diff
@@ -4,20 +4,53 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - '**.md'
   pull_request:
     types:
       - opened
       - synchronize
-    paths-ignore:
-      - '**.md'
```

Comment:
We can use conditions for Ruby jobs later, but there is no simple way with current GH Actions syntax:
* As I understand, `if changed_files != 'markdown'` will skip Ruby job for commits with Ruby files **and** Markdown (or other non-related) files.
* But enumeration of all related to Ruby jobs files can be huge.